src/components/project-details/project-details.tsx

```diff
@@ -5,19 +5,36 @@
 
 import { Project } from 'utils';
 import { linkPipe } from '../../shared/pipes';
-import './project-details.module.css';
+import styles from './project-details.module.css';
 
 interface ProjectDetailsProps {
   project: Project;
 }
 
 const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }: ProjectDetailsProps) => {
   return (
-    <div>
+    <div id={styles['project-details']} className="item-box">
+      <header>Project Details</header>
       <p>
-        Project Details Component: {project.wbsNum} - {project.name} in{' '}
-        {linkPipe(project.gDriveLink)}
+        Project Name: <span className={styles.important}>{project.name}</span>
+        WBS#{' '}
+        <span className={styles.important}>
+          {project.wbsNum.area}.{project.wbsNum.project}.{project.wbsNum.workPackage}
+        </span>
       </p>
+      <p>
+        Project Lead: <span className={styles.important}>{project.projectLead.firstName}</span>
+        Project Manager:{' '}
+        <span className={styles.important}>{project.projectManager.firstName}</span>
+      </p>
+      <p>
+        Duration: <span className={styles.important}></span>
+      </p>
+      <menu>
```

Comment:
Why an HTML menu tag? Quick Google search says that it has very little browser support so we likely don't want to be using it.

<img width="1141" alt="Screen Shot 2021-04-28 at 2 49 42 PM" src="https://user-images.githubusercontent.com/51571627/116477076-f9562100-a830-11eb-875d-e134d8a1aa50.png">